scripts/templates/params.hpp.mako

```diff
@@ -24,20 +24,20 @@ from templates import helper as th
 #include <bitset>
 
 namespace ${x}_params {
-
 template <typename T> inline void serializePtr(std::ostream &os, T *ptr);
+} // namespace ${x}_params
```

Comment:
I initially added the namespace because I wanted to implement these operators for primitive types, but ultimately decided against it in favor of the `serializePtr` function. And I left the namespace there because I thought it might be sometimes useful to selectively opt-in into the operators with `using namespace ...`.
I don't have a strong opinion on this, so I'm fine with the change, but there are some follow-up changes that this makes possible (e.g., no need to forward declare functions). I can do that in another PR.

Reply:
👍